app/backend/src/couchers/servicers/conversations.py

```diff
@@ -618,6 +621,53 @@ def InviteToGroupChat(self, request, context):
 
         return empty_pb2.Empty()
 
+    def RemoveGroupChatUser(self, request, context):
+        """
+        1. Get admin info and check it's correct
+        2. Get user data, check it's correct and remove user
+        """
+        with session_scope() as session:
+            # Admin info
+            your_subscription = (
+                session.query(GroupChatSubscription)
+                    .filter(GroupChatSubscription.group_chat_id == request.group_chat_id)
+                    .filter(GroupChatSubscription.user_id == context.user_id)
+                    .filter(GroupChatSubscription.left == None)
+                    .one_or_none()
+            )
+
+            # if user info is missing
+            if not your_subscription:
+                context.abort(grpc.StatusCode.NOT_FOUND, errors.CHAT_NOT_FOUND)
+
+            # if user not admin
+            if your_subscription.role != GroupChatRole.admin:
+                context.abort(grpc.StatusCode.PERMISSION_DENIED, errors.ONLY_ADMIN_CAN_MAKE_ADMIN)
```

Comment:
Please make new error strings in `couchers.errors` and use them here, these don't match the actual errors.

(Also two more instances below)